ERROR - 2023-11-08 08:22:39 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp81\htdocs\ci4\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp81\htdocs\ci4\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'antrian', 3306, '', 0)
#1 C:\xampp81\htdocs\ci4\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp81\htdocs\ci4\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp81\htdocs\ci4\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp81\htdocs\ci4\app\Helpers\site_helper.php(38): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp81\htdocs\ci4\app\Views\users\login.php(37): SiteHelper::instansi()
#6 C:\xampp81\htdocs\ci4\system\View\View.php(228): include('C:\\xampp81\\htdo...')
#7 C:\xampp81\htdocs\ci4\system\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#8 C:\xampp81\htdocs\ci4\system\Common.php(1190): CodeIgniter\View\View->render('users/login', Array, true)
#9 C:\xampp81\htdocs\ci4\app\Controllers\Users.php(27): view('users/login', Array)
#10 C:\xampp81\htdocs\ci4\system\CodeIgniter.php(942): App\Controllers\Users->Index()
#11 C:\xampp81\htdocs\ci4\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\xampp81\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp81\htdocs\ci4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#14 C:\xampp81\htdocs\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp81\\htdo...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp81\htdocs\ci4\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp81\htdocs\ci4\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp81\htdocs\ci4\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp81\htdocs\ci4\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp81\htdocs\ci4\app\Helpers\site_helper.php(38): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp81\htdocs\ci4\app\Views\users\login.php(37): SiteHelper::instansi()
#5 C:\xampp81\htdocs\ci4\system\View\View.php(228): include('C:\\xampp81\\htdo...')
#6 C:\xampp81\htdocs\ci4\system\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 C:\xampp81\htdocs\ci4\system\Common.php(1190): CodeIgniter\View\View->render('users/login', Array, true)
#8 C:\xampp81\htdocs\ci4\app\Controllers\Users.php(27): view('users/login', Array)
#9 C:\xampp81\htdocs\ci4\system\CodeIgniter.php(942): App\Controllers\Users->Index()
#10 C:\xampp81\htdocs\ci4\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#11 C:\xampp81\htdocs\ci4\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp81\htdocs\ci4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp81\htdocs\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp81\\htdo...')
#14 {main}
CRITICAL - 2023-11-08 08:22:39 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `instansi`
WHERE `id` = :id:', [...], false)
 3 APPPATH\Helpers\site_helper.php(38): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Views\users\login.php(37): SiteHelper::instansi()
 5 SYSTEMPATH\View\View.php(228): include('C:\\xampp81\\htdocs\\ci4\\app\\Views\\users\\login.php')
 6 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('users/login', [], true)
 8 APPPATH\Controllers\Users.php(27): view('users/login', [...])
 9 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Users->Index()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp81\\htdocs\\ci4\\public\\index.php')
